core/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.aerogear.android.core" />
+    package="org.aerogear.android.core">
+    <application>
+        <provider
+            android:name="org.aerogear.mobile.core.bootstrap.AeroGearBootstrap"
```

Comment:
replace with `{applicationId}`

Reply:
I can't use `{applicationId}` here or the class will not be found. This trick was used on the `authorities` only because Android doesn't allow have more than one authorities in the same app, and we wanna provide it automatically ;)

Is that makes sense?

Reply:
Tried that now. You are right. Sorry for problems.